composer.json

```diff
@@ -1,19 +1,23 @@
 {
     "name": "acquia/drupal-recommended-settings",
-    "type": "composer-plugin",
     "description": "The composer plugin for adding drupal-recommended-settings for Acquia Cloud.",
+    "license": "GPL-2.0-only",
+    "type": "composer-plugin",
     "keywords": [
         "drupal",
         "acquia",
         "drupal-recommended-settings"
     ],
-    "license": "GPL-2.0-only",
```

Comment:
This is required.

Reply:
yes that key is available , its not removed. Its just normalized